### PR TITLE
Added flight_computer.py and FlightComputer class

### PR DESCRIPTION
**Pull request by JackCotter**
Added flight_computer.py and FlightComputer class so that functions can be easily accessed in main

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,181 +1,11 @@
...
-            test_mode = True
-        else:
-            # NOTE: We explicitely DO NOT fail here because this is a rocket.
-            #       An incorrect argument is not a reason to fail completely.
-            print("Warning: Unkown argument supplied: ", arg)
-
-    return (test_mode)
-
+    flight_computer = FlightComputer()
+    flight_computer.startup()
```
Comment (lisaundre) [inline src/main.py:5]: The FlightComputer init() also runs the startup function, we don't need it twice

File: src/flight_computer.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        return np.sqrt(np.dot(v, v))
+
+    def __powered_flight(self):
+        time_at_start = time.time()
+        
+        while True:
+            self.rocket_data.refresh()
+            self.rocket_data.send_to_blackbox()
+            # TODO: send data to airbrakes
+            if time.time() > (time_at_start + 5):
```
Comment (klemie) [inline src/flight_computer.py:90]: Make these constants or global no magic number bb

File: src/flight_computer.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                break
+            elif self.rocket_data.current_altitude < -9:
```
Comment (klemie) [inline src/flight_computer.py:92]: same thing with this -9

Comment (klemie) [reply src/flight_computer.py:92]: how does the altitude be negative do we have a drill

File: src/flight_computer.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                break
+
+    def __coast_flight(self):
+        time_at_start = time.time()
+        
+        while True:
+            self.rocket_data.refresh()
+            self.rocket_data.send_to_blackbox()
+            # TODO: send data to airbrakes
+            if time.time > (time_at_start + 480):
```
Comment (klemie) [inline src/flight_computer.py:102]: magic number (480)

File: src/flight_computer.py (hunk 3 above)
Comment (klemie) [reply src/flight_computer.py:92]: current acceleration?

File: src/flight_computer.py [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            elif self.rocket_data.velocity < 0:
+                break
+        
+    def __recovery_flight(self):
+        time_at_start = time.time()
+        
+        while True:
+            self.rocket_data.refresh()
+            self.rocket_data.send_to_blackbox()
+            if time.time > (time_at_start + 300):
```
Comment (klemie) [inline src/flight_computer.py:113]: 300 is magic

File: src/flight_computer.py (hunk 2 above)
Comment (klemie) [reply src/flight_computer.py:90]: change to powered flight timeout

**Review by klemie — CHANGES_REQUESTED** [review]
🥲

File: src/flight_computer.py [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+# i2c = busio.I2C(board.SCL, board.SDA)
+
+LAUNCH_ACCELERATION_THRESHOLD = 10
+# TODO: complete filepath
+BLACKBOX_FILEPATH = "/media/pi/.."
+
+class FlightComputer:
+
+    def __init__(self):
+        self.rocket_data = RocketData()
```
[the remark this answers is not in the document]
Comment (JackCotter) [reply src/flight_computer.py:28]: @macsyd made a pull request. resolving.

**Review by klemie — APPROVED** [review]

**Pull request merged**